fenix/plugins/fenixdependencies/src/main/java/FenixDependenciesPlugin.kt

```diff
@@ -63,6 +65,8 @@ object FenixDependencies {
     const val androidx_transition = "androidx.transition:transition:${FenixVersions.androidx_transition}"
     const val androidx_datastore = "androidx.datastore:datastore:${FenixVersions.androidx_datastore}"
 
+    const val coil_compose = "io.coil-kt:coil-compose:${FenixVersions.coil_compose}"
```

Comment:
Let's just double check with somebody that the Coil license is acceptable usage in our project. See https://mozilla.slack.com/archives/G01EU5QU35Y/p1697732158673379?thread_ts=1697723194.283109&cid=G01EU5QU35Y

Reply:
To close this loop, Coil is no longer included in this PR